src/app/radio/radio-examples/radio-examples.component.tpl

```diff
@@ -0,0 +1,25 @@
+{% set html %}
+  {% include "../radio-example/radio-example.component.html" %}
+{% endset %}
+
+{% set typescript %}
+  {% include "../radio-example/radio-example.component.ts" %}
+{% endset %}
+
+{% set htmlChecked %}
+  {% include "../radio-checked-example/radio-checked-example.component.html" %}
+{% endset %}
+
+{% set typescriptChecked %}
+  {% include "../radio-checked-example/radio-checked-example.component.ts" %}
+{% endset %}
+
+<it-radio-example></it-radio-example>
+
+<it-source-display html="{$ html | replace("\{\{", "/\{/\{") | replace("\}\}", "/\}/\}") $}" typescript="{$ typescript | replace("\{\{", "/\{/\{") | replace("\}\}", "/\}/\}") $}" >
```

Comment:
Solo un _bad smell_: tutta la parte che sta attorno all'`it-source-display` mi sembra poco comprensibile e manutenibile a causa di passaggi di encoding complesso. Sarebbe da trovare un modo più chiaro di trattare la cosa.